setup.py

```diff
@@ -93,20 +93,19 @@ def build_extension(self, ext: CMakeExtension):
                 configure_args += []
             else:
                 configure_args += ["-DENABLE_OPENMP=OFF"]
-        elif platform.system() == "Linux":
-            if platform.machine() == "x86_64":
-                configure_args += ["-DENABLE_AVX=ON"]  # Enable AVX if x64 on Linux
         elif platform.system() == "Windows":
             configure_args += ["-DENABLE_OPENMP=OFF", "-DENABLE_BLAS=OFF"]
+            if not debug:
+                build_args += ["--config", "RelWithDebInfo"]
```

Comment:
Should we set this at the outer package level too? As in, make it default behaviour for all platforms?

Reply:
We don't need it, but I can do it.